stable/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 8.15.0
+version: 8.15.1
```

Comment:
Bump the MINOR version could make sense. `8.xx.0`

Reply:
@Aisuko I have resolved the conflict and bumped the minor version, please review :)